README.md

```diff
@@ -77,13 +95,15 @@ a model will be trained using the provided example config file and xyz files (ca
 
 We recommend installing uv for dependency management when developing for `aiida-mlip`:
 
+
 1. Install [uv](https://docs.astral.sh/uv/getting-started/installation)
-2. Install `aiida-mlip` with dependencies in a virtual environment:
+2. Setup [PostgreSQL](https://aiida.readthedocs.io/projects/aiida-core/en/stable/installation/guide_complete.html#core-psql-dos)
+3. Install `aiida-mlip` with dependencies in a virtual environment:
```

Comment:
I'd probably either break these up or reword the lead-in sentence, as this essentially suggests this is all related to `uv`.